TicTacToeApp/ViewController.swift

```diff
@@ -8,18 +8,38 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+public class ViewController: UIViewController {
+  var currentPlayerMark = "X"
```

Comment:
Yeah, it actually does do that now. Though the comment still is applicable to other variables that are initialized similarly.